modules/labs-react/pagination/lib/Pagination/Controls.tsx

```diff
@@ -62,8 +63,8 @@ export const StepToPreviousButton = ({onClick, model, ...restProps}: ControlButt
   return (
     <IconButton
       aria-disabled={isDisabled || undefined}
-      variant={IconButton.Variant.Square}
-      size={IconButton.Size.Small}
+      variant={'square'}
```

Comment:
@NicholasBoll why do some of these turn into `variant="square"` (line 43) and others `variant={'square'}`?

Reply:
The codemod does `{"square"}` since I'm really adjusting a `MemberExpression` to a `Literal`, but the node is still inside a `JSXExpressionContainer`. It is a stylistic preference to turn them into `JSXLiteral` instead. I was doing this manually with a replace-all regex post-processing. It probably happened as a result of a file being saved (turning `"` into `'`) before the Regex was run. I'll search for these patterns and adjust them for consistency.

Reply:
But these are both the same context: returning a JSX element with `variant={IconButton.Variant.Square}`. The codemod has updated both, but one gets `"square"`, the other gets `{'square'}` - the inconsistency is what I'm wondering about.

Reply:
Fixed